service/src/main/java/eu/europa/ec/fisheries/ers/service/bean/FishingTripServiceBean.java

```diff
@@ -634,6 +640,31 @@ FishingTripResponse buildFishingTripSearchRespose(Set<FishingTripId> fishingTrip
         return response;
     }
 
+    private CronologyDTO buildCronologyDTO(String tripId) throws ServiceException {
+        CronologyDTO cronologyDTO = new CronologyDTO();
+        FishingActivityQuery query = new FishingActivityQuery();
+        Map<SearchFilter, String> searchCriteriaMap = new EnumMap<>(SearchFilter.class);
+        searchCriteriaMap.put(SearchFilter.TRIP_ID, tripId);
+        query.setSearchCriteriaMap(searchCriteriaMap);
+        SortKey sortKey = new SortKey();
+        sortKey.setSortBy(SearchFilter.PERIOD_START); // this is important to find out first and last fishing activity for the Fishing Trip
+        sortKey.setReversed(false);
+        query.setSorting(sortKey);
+        List<FishingActivityEntity> fishingActivityEntityList = fishingActivityDao.getFishingActivityListByQuery(query);
+
+        Date startDate = fishingActivityEntityList.stream()
+                .findFirst()
+                .map(FishingActivityEntity::getCalculatedStartTime)
+                .orElse(null);

```

Comment:
You can apply DateUtils.dateToString here, i.e.:

    String formattedStartDate = fishingActivityEntityList.stream()
                .findFirst()
                .map(FishingActivityEntity::getCalculatedStartTime)
                .map(DateUtils::dateToString)
                .orElse(null);